src/app/workspace/resource/resource.component.html

```diff
@@ -9,6 +9,7 @@
                 [images]="representationsToDisplay"
                 [imageCaption]="(incomingResource ? resource.res.label + ': ' + incomingResource.res.label : resource.res.label)"
                 [compoundNavigation]="compoundPosition" (goToPage)="compoundNavigation($event)"
+                             [resourceIri]="incomingResource ? incomingResource.res.id : resource.res.id" [project]="resource.res.attachedToProject"
```

Comment:
Just a tiny detail. The indention is wrong resp. could you place each property on a separate line. The `(goToPage)` should be the last one. We defined (somewhere) the rule that we have first the normal parameters followed by the events. But as I said it's just a detail. Otherwise it looks good.

Reply:
Thanks! Done.